plexus-compilers/plexus-compiler-javac/src/main/java/org/codehaus/plexus/compiler/javac/JavacCompiler.java

```diff
@@ -343,8 +343,15 @@ protected static boolean isJava16()
         if ( !config.isShowWarnings() )
         {
             args.add( "-nowarn" );
+        } else
+        {
+            String warnings = config.getWarnings();
+            if (null != warnings && !StringUtils.isEmpty(warnings))
```

Comment:
no need of null check if you use `StringUtils.isEmpty`

Reply:
> no need of null check if you use `StringUtils.isEmpty`

no need of negation if you use `StringUtils.isNotEmpty`